meta-luneos/recipes-webos-ose/maliit-framework-webos/files/maliit-server.service

```diff
@@ -15,16 +15,18 @@
 # SPDX-License-Identifier: Apache-2.0
 
 [Unit]
-Description=webos - "%n"
-Requires=ls-hubd.service surface-manager.service
-After=ls-hubd.service surface-manager.service
+Description=meta-webos - "%n"
+ConditionPathExists=/tmp/lsm-ready
+Requires=ls-hubd.service settings-service.service surface-manager.service
+After=ls-hubd.service settings-service.service surface-manager.service
```

Comment:
There was a fix I made some month ago to fix maliit's restart, which happens when upgrading LSM package for instance. See https://github.com/webOS-ports/meta-webos-ports/commit/28ee326aa41e1641a2aec899d5f70c3aa043dd1c for the main fix:

Fix two things:
* maliit-server.path triggers maliit-server.service when lsm-ready
  file is present. However, as LSM depends on maliit-server, it
  means we'll never be able to stop the surface-manager service.
  In addition, maliit-server's startup is already triggered by
  surface-manager.service, so it's useless.
* having maliit-server@.service in SYSTEMD_SERVICE will insert it
  in the postrm scripts of the package. However, this is a
  template service, so it's incorrect.
  Similarly, starting it at startup is incorrect as the instance
  argument is mandatory.
  So just remove it, maliit-server@0.service will be started by
  maliit-server.service anyway.